### ISSUE TITLE
Select PLA Cruise Missiles (CJ-10, YJ-12, YJ-18)

### ISSUE THREAD

**Issue by RickDNiu** [PygmalionOfCyprus/cmo-db-requests#5084]
### CJ-10 

- **CJ-10,** aka DH-10, DF-10, GLCM, 2006

2.5t, 1.7t wo SRB, 8.3m length.

Target: Ground.

Platform: Mobile launcher (3x each).

Range and propulsion: 2500km, Mach 0.75 at 160ft, turbofan.

Warhead: 500kg HE/80kt nuclear.

GNC: INS + Beidou, TERCOM, support waypoints, pre-briefed only, 10m CEP.

> Beidou was still premature in 2006. 

- **CJ-10A**, IIR+DL, GLCM, 2015

~2.5t, 1.7t wo SRB, 8.3m length.

Target: Ground/Surface/Submarine.

Platform: Mobile launcher (3x each).

Range and propulsion: 2500km, Mach 0.75 at 100ft, turbofan.

Warhead: 500kg HE/350kg Cluster munition/350kg penetrator/80kt nuclear. 

GNC: INS + Beidou, TERCOM, support waypoints, IIR seeker, two way DL, 5m CEP, BOL.

- **A/KD-20**, aka CJ-20, CJ-10A, ALCM, IIR+DL, 2015

~1.7t.

Target: Ground.

Platform: Air.

Range and propulsion: 2500km, Mach 0.75 at 100ft, turbofan.

Warhead: 500kg HE/350kg Cluster munition/350kg penetrator/80kt nuclear. 

GNC: INS + Beidou, TERCOM, support waypoints, IIR seeker, two way DL, 5m CEP, BOL.

- **YJ-100**, ASCM, IIR+ARH+DL, CJ-10A mod, 2015

~2.5t, 1.7t wo SRB, 8.3m length.

Target: Surface/Ground.

Platform: Air/Ground/Surface/Submarine.

Range and propulsion: 2500km (~1400nm?), Mach 0.75 at 60ft, turbofan.

Warhead: 500kg SAP.

GNC: INS + Beidou, TERCOM, support waypoints, ARH+IIR seeker, may have passive radar seeker, two way DL, 2m CEP, BOL.

---

### YJ-12

**YJ-12, YJ-12A, YJ-12B, CM-302**, ASCM, 2009

Target: Surface/Ground(?).

Platform: Air(JH-7, H-6, J-16)/Ground(B)/Surface(A).

Range and propulsion: 500km, 250km export, ~Mach 2.5 cruise, up to Mach 4 terminal, ramjet.

Warhead: 300kg SAP.

GNC: INS + Beidou, TERCOM, support waypoints, ARH + passive radar seeker, 2m CEP, BOL.

---

### YJ-18

**YJ-18/YJ-18A**, CH-SS-NX-13, 2014

Platform: Ground/Surface/Submarine(A).

Range and propulsion: 656km, ~Mach 0.8 cruise, turbojet, up to Mach 3 terminal, SRM.

Warhead: 300kg SAP.

GNC: INS + Beidou, TERCOM, support waypoints, ARH + passive radar seeker, 2m CEP, one way DL during cruise, BOL.

**Comment by PygmalionOfCyprus** [comment]
Do we have sources for this stuff?

**Comment by RickDNiu** [comment]
Yes. Generally aggregation of public information, public displays, basing and deployment, propaganda materials etc. Will include some later but could be difficult to find all the originals

**Comment by PygmalionOfCyprus** [comment]
Sure, I figured you weren't just making it up on the spot. 😆 

A lot of this info I saw but intentionally determined to be unreliable or unverified when I was doing my review of PLARF weapons. A few points for debate:

- I'm not convinced that the CJ-10 is in fact capable of carrying a nuclear warhead. This feels like an assumption people made based on the fact that it's the Sino-Tomahawk, but to my knowledge, no official/official-adjacent Chinese source has mentioned anything of the sort, right? There's a lot of fuzz/skepticism in the analyst community on this AFAIK.
- Likewise the cluster/penetrator warheads for the CJ-10 variants. This makes sense, so I don't doubt it outright, I'd just love to see the source for this specifically to verify.
- I also can't find any concrete proof of the IIR seeker on the CJ-10A, so I wasn't sure if I wanted to go that far and compromised on just reducing CEP. I actually made a note of it in the DB manager notes: 
![image](https://github.com/user-attachments/assets/fd064d1d-754c-4bd1-bfb7-097b9759402f)

YJ-12s and YJ-18 stuff seems pretty unobjectionable.
